backend/dal/Data/Configuration/AddressConfiguration.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Pims.Dal.Data.Entities;
+
+namespace Pims.Dal.Data.Configuration
```

Comment:
I wouldn't keep the `Data` folder.  Everything in `Data` should be moved to the root of the `Dal` project.  You'll need to update all your references to this namespace.